config/notifiers.go

```diff
@@ -109,9 +109,6 @@ var (
 			VSendResolved: false,
 		},
 		Message: `{{ template "wechat.default.message" . }}`,
-		ToUser:  `{{ template "wechat.default.to_user" . }}`,
```

Comment:
no need to remove those lines

Reply:
> no need to remove those lines

if don't remove default value。I can’t check the ToUser/ToTag/ToParty if them is empty!
unless  I use tmplText in UnmarshalYAML to get real value，it‘s not good。

on the other side,  I think there is no need to configure the template inside, it's better that all configurations come from the same file，What is your opinion?

Reply:
Right, sorry for confusing you. Lets just check that `MessageType` matches the allowed values.